public/js/editor.js

```diff
@@ -23,12 +23,16 @@ pimcore.bundle.tinymce.editor = Class.create({
     },
 
     initializeWysiwyg: function (e) {
+
         if (e.detail.context === 'object') {
             if (!isNaN(e.detail.config.maxCharacters) && e.detail.config.maxCharacters > 0) {
                 this.maxChars = e.detail.config.maxCharacters;
-            }else{
+            } else {
                 this.maxChars = -1;
             }
+            e.detail.config = Object.assign({}, e.detail.config, {ui_mode: 'combined'});
+        } else {
+            e.detail.config = Object.assign({}, e.detail.config, {ui_mode: 'split'});
```

Comment:
```suggestion
            e.detail.config = Object.assign({}, {ui_mode: 'combined'}, e.detail.config);
        } else {
            e.detail.config = Object.assign({}, {ui_mode: 'split'}, e.detail.config);
```
@kingjia90 I think the event config should have higher priority than the internal logic. WDYT?